Docs/production-deployment/example/applications/example-app/Dockerfile

```diff
@@ -0,0 +1,22 @@
+# syntax=docker/dockerfile:1
+FROM alpine:3.19 AS unpacker
+
+RUN apk add --no-cache unzip
+# Copy the zipped artifacts from Maven build output
+COPY target/zips/ /tmp/zips/
+
+# Unpack each zip into its own folder
+RUN mkdir -p /tmp/projects && \
+    for file in /tmp/zips/*.zip; do \
+        name=$(basename "$file" .zip); \
+        mkdir -p "/tmp/projects/$name"; \
+        unzip -o "$file" -d "/tmp/projects/$name"; \
+    done
+# Use the predefined image from Docker Hub as the base
```

Comment:
There no needs to unpack zip files. `openltablets/ws` works with zip files out of the box.